index.html

```diff
@@ -5,9 +5,218 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./resources/css/main.css" />
-    <title>Template</title>
+    <script
+      src="https://kit.fontawesome.com/192b6fe7c2.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <title>Almanac Festival</title>
   </head>
   <body>
-    <h1>Template</h1>
+    <header>
+      <nav class="nav__layer">
+        <ul class="nav__layer--list">
+          <li class="li"><i class="fa-brands fa-facebook"></i></li>
+          <li class="li"><i class="fa-brands fa-twitter"></i></li>
+          <li class="li">English</li>
+          <li class="li">My Page</li>
+          <li class="li">Log out</li>
+        </ul>
+      </nav>
+      <nav>
+        <div class="desktop__menu-container">
+          <div class="desktop__menu--image">
+            <img src="./resources/img/menulogo.png" alt="" />
+          </div>
+          <div class="desktop__menu">
+            <ul class="desktop__menu--list">
+              <li class="link-1">
+                <a href="about.html">About</a>
+              </li>
+
+              <li class="link-2"><a href="#">Program</a></li>
+
+              <li class="link-3"><a href="#">Join</a></li>
+              <li class="link-4"><a href="#">Sponsors</a></li>
+              <li class="link-5"><a href="#">News</a></li>
+              <li class="almanac"><a href="#">Almanac</a></li>
+            </ul>
```

Comment:
- [ ] please make sure that the navigation links are working in the mobile menu 
![image](https://user-images.githubusercontent.com/61790587/175362819-0e7a4f18-c03a-466d-8ee3-e8cebc03ba23.png)